File: src/web/btg-app/src/app/models/response-result.ts
export class BadRequestResult {
    error: ResponseResult;
  
    constructor(){
      this.error = new ResponseResult();
    }
  }
  
  export class ResponseResult {
    title: string;
    status: number;
    errors: ReponseErroMessages;
  
    constructor() {
      this.errors = new ReponseErroMessages();
    }
  }
  
  export class ReponseErroMessages {
    Mensagens: string[] = [];
  
  }
  

File: src/building blocks/BTG.Core/Communication/ResponseResult.cs
﻿namespace BTG.Core.Communication
{
    public class ResponseResult
    {
        public ResponseResult()
        {
            Errors = new ReponseErroMessages();
        }

        public string Title { get; set; }
        public int Status { get; set; }
        public ReponseErroMessages Errors { get; set; }
    }

    public class ReponseErroMessages
    {
        public ReponseErroMessages()
        {
            Mensagens = new List<string>();
        }

        public List<string> Mensagens { get; set; }
    }
}
